vowpalwabbit/cb_adf.cc

```diff
@@ -367,7 +369,7 @@ void output_example(vw& all, cb_adf& c, example& ec, v_array<example*>* ec_seq)
 
   float loss = 0.;
 
-  uint32_t action = ec.pred.multiclass;
+  uint32_t action = ec.pred.a_s[0].action;
```

Comment:
lacking a bit of context on pred.a_s, but is it always guaranteed to contain at least 1 item?